### PR TITLE
`Programming exercises`: Add information to feedback view

### PR DESCRIPTION
**Pull request by simon-hng**
#### General

- [x] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
#### Client
- [x] I followed the [coding and design guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client/) and ensured that the layout is responsive.
- [x] I added multiple screenshots/screencasts of my UI changes.

### Motivation and Context

Students should know how many feedbacks they have in each category.

### Description
- Adds the amount of feedbacks per group as a number after the name `Wrong -> Wrong (3)`
- For feedbacks: Adds a gap between the amount of points and the feedback title 
- Adds a tooltip on hover for the date in the submission information (see screenshot below)

### Steps for Testing

Prerequisites:
Go to a feedback detail view and check if there are any issues and if it looks like in the screenshots

### Screenshots

- Amount of feedback next to category name (Dont mind the blue background. I accidentally selected the points)
Screenshot_20230109_163208

- The purple dotted line between title and amount of points is the minimal padding
Screenshot_20230110_123730

- Date tooltip on hover
Screenshot_20230110_125439

### CODE REVIEW

**Review by JohannesStoehr — APPROVED** [review]
Code changes and screenshots look good to me

**Review by Strohgelaender — DISMISSED** [review]
code 👍

File: src/main/webapp/app/exercises/shared/feedback/node/feedback-node.component.html [hunk 1]
```diff
@@ -18,9 +18,7 @@ <h4 class="feedback-item__category">{{ feedbackItem.name }} {{ feedbackItem.titl
 <details *ngIf="feedbackItemGroup" class="feedback-item-group" [open]="feedbackItemGroup.open">
     <summary class="feedback-item-group__summary" [ngClass]="'bg-' + feedbackItemGroup.color">
         <div class="feedback-item-group__summary__content">
-            <h4 class="feedback-item__category">
-                {{ feedbackItemGroup.name | titlecase }}
-            </h4>
+            <h4 class="feedback-item__category">{{ feedbackItemGroup.name | titlecase }} ({{ feedbackItemGroup.members.length }})</h4>
```
Comment (ge65cer) [inline src/main/webapp/app/exercises/shared/feedback/node/feedback-node.component.html:21]: I'd propose using [Badges](https://getbootstrap.com/docs/5.3/components/badge).

Comment (simon-hng) [reply src/main/webapp/app/exercises/shared/feedback/node/feedback-node.component.html:21]: In general this seems like a good idea but I believe it's gonna get confusing as the background is already colored. I will try this out later and add a screenshot here

Comment (ge65cer) [reply src/main/webapp/app/exercises/shared/feedback/node/feedback-node.component.html:21]: `text-bg-light` or `text-bg-dark` maybe?

Comment (simon-hng) [reply src/main/webapp/app/exercises/shared/feedback/node/feedback-node.component.html:21]: Screenshot_20230110_234530

File: src/main/webapp/app/exercises/shared/feedback/feedback.component.html [hunk 2]
```diff
@@ -44,7 +44,7 @@ <h4>
                         </ng-template>
                     </h4>
 
-                    <span class="badge" [ngClass]="badge.class" id="result-score-badge" ngbTooltip="{{ badge.tooltip | artemisTranslate }}">
+                    <span *ngIf="badge" class="badge" [ngClass]="badge.class" id="result-score-badge" ngbTooltip="{{ badge.tooltip | artemisTranslate }}">
```
Comment (simon-hng) [inline src/main/webapp/app/exercises/shared/feedback/feedback.component.html:47]: this was a issue, that actually does not have anything to do directly with this PR but that I noticed and quick-fixed.

I will open a follow-up PR that deals with these status badges in general and investigate this further

**Review by Strohgelaender — APPROVED** [review]

File: src/main/webapp/app/exercises/shared/feedback/node/feedback-node.component.html (hunk 1 above)
Comment (ge65cer) [reply src/main/webapp/app/exercises/shared/feedback/node/feedback-node.component.html:21]: Fourth option looks reasonable.

**Pull request merged**